integration/spark2/src/main/java/org/apache/carbondata/spark/vectorreader/VectorizedCarbonRecordReader.java

```diff
@@ -121,12 +126,24 @@ public VectorizedCarbonRecordReader(QueryModel queryModel, InputMetricsStats inp
       while (ext != null) {
         if (ext instanceof FileNotFoundException) {
           throw new InterruptedException(
-              e.getMessage() + ". insert overwrite may be in progress.Please check");
+              "Insert overwrite may be in progress.Please check " + e.getMessage());
         }
         ext = ext.getCause();
       }
       throw new InterruptedException(e.getMessage());
     }
+    catch (Exception e) {
```

Comment:
This catch handling is the same as previous catch?

Reply:
this catch block is for handling any checked exception if its thrown in the form of Runtime exception . In this case FileNotFoundException is thrown by casting as Runtime exception.